api/topic_private_test.go

```diff
@@ -0,0 +1,272 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/ONSdigital/dp-topic-api/apierrors"
+	"github.com/ONSdigital/dp-topic-api/config"
+	"github.com/ONSdigital/dp-topic-api/models"
+	storeMock "github.com/ONSdigital/dp-topic-api/store/datastoretest"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestGetTopicPrivateHandler(t *testing.T) {
+
```

Comment:
(suggestion): I can you can run these tests in parallel ? ```t.Parallel``` ?

Reply:
Done